game/src/contrib/utils/components/ai/AITools.java

```diff
@@ -136,7 +141,8 @@ public static GraphPath<Tile> calculatePath(final Coordinate from, final Coordin
      */
     public static GraphPath<Tile> calculatePathToRandomTileInRange(
             final Point point, final float radius) {
-        Coordinate newPosition = randomAccessibleTileCoordinateInRange(point, radius);
+        Coordinate newPosition =
+                randomAccessibleTileCoordinateInRange(point, radius).orElse(point.toCoordinate());
```

Comment:
@AMatutat API: Warum muss ich für eine Kalkulation eines Pfades Koordinaten nutzen? Wäre es nicht viel natürlicher (und hier besser lesbar), wenn wir stattdessen Tiles erlauben? Also `calculatePath(Tile from, Tile to)` stattdessen (oder zusätzlich)? Dann kann man hier auf das ewige Rausprökeln der Coordinate (`.toCoordinate()`) verzichten ...